RZAssemblage/Private/RZPropertyAssemblage.m

```diff
@@ -0,0 +1,122 @@
+//
+//  RZPropertyAssemblage.m
+//  RZAssemblage
+//
+//  Created by Brian King on 3/21/15.
+//  Copyright (c) 2015 Raizlabs. All rights reserved.
+//
+
+#import "RZPropertyAssemblage.h"
+#import "RZAssemblage+Private.h"
+
+static char RZPropertyContext;
```

Comment:
Simpler to use `static void* const RZPropertyContext = (void *)&RZPropertyContext`